crates/router/src/connector/bluesnap/transformers.rs

```diff
@@ -590,6 +621,14 @@ impl TryFrom<&BluesnapRouterData<&types::PaymentsCompleteAuthorizeRouterData>>
             .parse_value("BluesnapRedirectionResponse")
             .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
 
+        let transaction_meta_data = match item.router_data.request.metadata.as_ref() {
+            Some(metadata) => Some(BluesnapMetadata {
+                meta_data: Vec::<RequestMetadata>::foreign_from(metadata.peek().to_owned()),
```

Comment:
Incase of manual capture and void were we able to see the metadata for those transaction?

Reply:
Yes, metadata sent in auth is visible on dashboard for capture and void calls